model/accounts_mgmt/v1/labels_resource.model

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages the collection of labels.
+resource Labels {
```

Comment:
Is this a generic read-only list of _all_ labels (regardless or whether they belong to an Account, Organization, or Subscription)? Wondering since there are no other methods defined.

Reply:
Exactly, all labels

Reply:
I like it